recipes/openssl/1.x.x/conanfile.py

```diff
@@ -436,9 +436,15 @@ def _get_env_build(self):
             self._env_build = AutoToolsBuildEnvironment(self)
         return self._env_build
 
+    def _get_default_openssl_dir(self):
+        if self.settings.os == "Linux" and self._full_version >= "1.1.0":
+            return "/etc/ssl"
+        return os.path.join(self.package_folder, "res")
```

Comment:
I actually think, that you should follow the default value for all versions, even if it works as it is.

Reply:
Nice, thanks :)

Reply:
Ok, thanks for the explanation 👍